### PR TITLE
[BPK-4331] Migrate more pages to mdx

### PR DESCRIPTION
**Pull request by shaundon**
scrollable calendar doesn't yet work, so I've got the change in there commented out, we just need to change it when it's fixed on the web side.

### CODE REVIEW

File: docs/src/pages/components/ScrollableCalendar/ScrollableCalendarPage.js [hunk 1]
```diff
@@ -35,7 +36,7 @@ const ScrollableCalendarPage = () => (
       </IntroBlurb>,
     ]}
     webSubpage={<Web wrapped />}
-    nativeSubpage={null}
+    // webSubpage={<MarkdownPage content={Web} {...webMetadata} />}
```
Comment (georgegillams) [inline docs/src/pages/components/ScrollableCalendar/ScrollableCalendarPage.js:39]: ¿Que es el problemo con esta?

Comment (georgegillams) [reply docs/src/pages/components/ScrollableCalendar/ScrollableCalendarPage.js:39]: Just saw the desciption - makes sense 👍

Comment (shaundon) [reply docs/src/pages/components/ScrollableCalendar/ScrollableCalendarPage.js:39]: Se queja de que `date-fns` no está disponible, por lo que debe agregarse como una dependencia en el paquete.

Comment (georgegillams) [reply docs/src/pages/components/ScrollableCalendar/ScrollableCalendarPage.js:39]: NGL my Spanish is not advanced enough for that response 😂

**Review by georgegillams — APPROVED** [review]

**Pull request merged**